vulture/core.py

```diff
@@ -184,13 +184,16 @@ def exclude(name):
             else:
                 self.scan(module_string, filename=module)
 
-            whitelist_names = ['stdlib.py']
-            for name in whitelist_names:
+            for name in self.defined_imports:
                 path = os.path.join('whitelists', name)
                 if exclude(path):
                     self.log('Excluded Whitelist:', path)
                 else:
-                    module_data = pkgutil.get_data('vulture', path)
+                    try:
+                        path += '.py'
+                        module_data = pkgutil.get_data('vulture', path)
+                    except IOError:
```

Comment:
OK. Can you add the comment `// We asked for a non-existent file. The error is expected.` above `pass`, please?